Dockerfile

```diff
@@ -18,17 +18,12 @@ COPY pkg ./pkg
 RUN make build
 
 FROM registry.svc.ci.openshift.org/ocp/4.6:cli as origincli
-
 FROM registry.svc.ci.openshift.org/ocp/4.6:base
-RUN INSTALL_PKGS=" \
-      openssl \
-      rsync \
-      " && \
-    yum install -y $INSTALL_PKGS && \
-    rpm -V $INSTALL_PKGS && \
-    yum clean all && \
-    mkdir /tmp/ocp-clo && \
-    chmod og+w /tmp/ocp-clo
+
+RUN dnf -v -y install openssl
```

Comment:
I suggest to keep `rsync` because it is a mandatory utility for must-gather. A similar dnf command to `yum clean all` should be also kept here for minimizing container size.

Reply:
It's already present in the base image, so installing it just causes "already installed" noise.
However I'll put it back in case we ever use a base image without it. +1 to the dnf clean.